src/types.db

```diff
@@ -148,6 +149,9 @@ node_octets             rx:DERIVE:0:U, tx:DERIVE:0:U
 node_rssi               value:GAUGE:0:255
 node_stat               value:DERIVE:0:U
 node_tx_rate            value:GAUGE:0:127
+nr_overcommit_hugepages value:DERIVE:0:U
+nr_hugepages            value:DERIVE:0:U
```

Comment:
Our policy is to report metrics like this in a way that they can be summed up. This is one of these cases where "total" and "free" / "reserved" are reported, which don't yield a useful sum. Could you please do something along the lines of `used = total - (free + reserved)` and then report the number of used hugepages?